charts/jobs/values.yaml

```diff
@@ -6,15 +35,15 @@ resources:
     cpu: 100m
     memory: 128Mi
 
-tasks: {}
-  # someTask:
-  #   enabled: true
-  #   shell: bash # default sh
-  #   env:
-  #     BLA: somebla
-  #   secret:
-  #     DIDA: somedida
-  #   type: job # cronjob
-  #   ttlSecondsAfterFinished: 86400
-  #   schedule: '1 * * * *'
-  #   script: echo "bla"
+runPolicy: Always # OnSpecChange
+# schedule: '1 * * * *'
+
+script: echo "bla"
+secrets: []
+  # - name: secret-name
+  #   entries:
+  #       - foo
+  #       - bar
+
+type: job # cronjob
```

Comment:
and coming to think of it, can we stick with CamelCasing and have enum with `Job` and `CronJob`?